lib/config/openGemini_dir.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 Huawei Cloud Computing Technologies Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+var homeDir = openGeminiDir()
+
+// openGeminiDir retrieves the openGemini directory.
+func openGeminiDir() string {
+	var dir string
+	// By default, store meta and data files in current users home directory
+	u, err := user.Current()
+	if err == nil {
+		dir = u.HomeDir
+	} else if home := os.Getenv("HOME"); home != "" {
+		dir = home
+	} else {
+		wd, err := os.Getwd()
+		if err != nil {
+			panic(err)
+		}
+		dir = wd
+	}
+	homeDir := filepath.ToSlash(dir)
```

Comment:
I suggest `homeDir` should contain `.openGemini`.

```suggestion
	homeDir := filepath.ToSlash(filepath.Join(dir, ".openGemini"))
```